lms/djangoapps/discussion/templates/discussion/edx_ace/response_notification/email/body.html

```diff
@@ -56,3 +56,7 @@
     </tr>
 </table>
 {% endblock %}
+
+{% block google_analytics_pixel %}
+<img src="{{ ga_pixel_url }}"/>
```

Comment:
I had to modify my markup non-trivially to make the sailthru pixel accessible, now it looks like this:

```
<img src="{{ ga_pixel_url }}" alt="" role="presentation" aria-hidden="true" />
```